sdk/ai/Azure.AI.Inference/src/Customized/ChatCompletionsClient.cs

```diff
@@ -43,6 +43,25 @@ public ChatCompletionsClient(Uri endpoint, AzureKeyCredential credential, AzureA
             _apiVersion = options.Version;
         }
 
+        /// <summary> Initializes a new instance of ChatCompletionsClient. </summary>
+        /// <param name="endpoint"> Service host. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="authorizationScopes"> The authorization scopes for token. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public ChatCompletionsClient(Uri endpoint, TokenCredential credential, string[] authorizationScopes, AzureAIInferenceClientOptions options=default)
```

Comment:
I am pretty sure the `string[] authorizationScopes` parameter is the root cause of the build analyze error.

The generator has a bug that it does properly handles the array type in customization code.
I think we need to fix it.